src/Bundle/AutoMapperBundle/Tests/ServiceInstantiationTest.php

```diff
@@ -38,7 +38,7 @@ public function testAutoMapper()
         self::assertSame('yolo', $userDto->getName());
         self::assertSame(13, $userDto->age);
         self::assertSame(((int) date('Y')) - 13, $userDto->yearOfBirth);
-        self::assertSame('fallback@foobar.org', $userDto->email);
```

Comment:
I don't like modifying a test, can't we replace it ?

Reply:
This test have already been modified in #450 (to test that both implementations works).
With this change, we're back to the test as it was before #450

Reply:
Ok, I see that https://github.com/janephp/janephp/pull/453/files#diff-4e2f71e5fd9f7f92d6ee85687e14b25f043c3a572d0b198ab299a9385b6cf6e4L41

Thanks !